src/main/java/org/jenkinsci/plugins/bigpanda/ChangesBuilder.java

```diff
@@ -177,18 +177,19 @@ private long getEndTime() {
     private String getBuildDescription() {
         Run<?, ?> bld = this.build;
         Job<?, ?> parent;
+        String description = "No Description Found";
 
         if (bld.getDescription() != null && bld.getDescription().trim().length() > 0) {
-            return bld.getDescription();
+            description = bld.getDescription();
         } else {
             parent = bld.getParent();
 
-            if (parent.getDescription() == null || parent.getDescription().trim().length() <= 0) {
-                return parent.getDescription();
-            } else {
-                return "No Description";
+            if (parent.getDescription() != null && parent.getDescription().trim().length() > 0) {
+                description = parent.getDescription();
             }
         }
+
+        return description;
```

Comment:
Any reason to track description as a new variable? Doesn't appear to be used again. Could simply be 3 return statements

Reply:
Two reasons: 
- When performing `mvn package` it actually complained about not having a return at the end
- I don't like the convention of having multiple return statements